### ISSUE TITLE
Problem: Unable to fetch levels of description from AtoM

### ISSUE THREAD

**Issue by sromkey** [artefactual/archivematica#866]
To reproduce:

From Administration/ AtoM, click on Levels of Description then on Fetch from AtoM. There are no errors shown but the levels do not appear. I tested this on the augustus test server.

**Comment by jraddaoui** [comment]
Ready for code review in #893

**Comment by jraddaoui** [comment]
Fix added to qa/1,x and stable/1.7.x

**Comment by nickwilkinson** [comment]
Reopening since @jraddaoui is still working on this.

**Comment by jraddaoui** [comment]
Update:

We fixed this issue by removing auth. in the endpoints were there wasn't in stable/1.7.x. But in https://github.com/artefactual/archivematica/pull/900 we re-added auth. to those endpoints and added session auth. to the entire dashboard API.

After that, we've realized that session auth. is failing in POST request, for example to '.../api/filesystem/metadata/' when a LoD is set in the 'Edit metadata' pop-out of the arrange panel.
